xarray/core/common.py

```diff
@@ -214,36 +227,38 @@ def _ipython_key_completions_(self):
         return list(set(item_lists))
 
 
-def get_squeeze_dims(xarray_obj, dim, axis=None):
+def get_squeeze_dims(xarray_obj, dim: Union[str, Tuple[str, ...], None] = None,
```

Comment:
Probably should be something more relaxed, perhaps `dim: Union[Hashable, Iterable[Hashable]]`

Reply:
trouble with Iterable is that it doesn't necessarily allow you to iterate upon it twice (e.g. a file handler). I've tweaked the code to allow for it.